Python/Product/PythonTools/PythonTools/Repl/PythonCommonInteractiveEvaluator.cs

```diff
@@ -401,6 +404,10 @@ internal abstract class PythonCommonInteractiveEvaluator :
 
             string candidate;
             if (!string.IsNullOrEmpty(displayName)) {
+                foreach (var c in Path.GetInvalidFileNameChars()) {
+                    displayName = displayName.Replace(c, '_');
```

Comment:
Is this better than simply removing the characters?

Reply:
I feel it's more readable.